gcp_variant_transforms/transforms/merge_header_definitions.py

```diff
@@ -14,127 +14,59 @@
-
-# ``Definition`` cherry-picks the attributes from vcf header definitions that
-# are critical for checking field compatibilities across VCF files.
-Definition = namedtuple('Definition',
-                        [vcf_header_io.VcfParserHeaderKeyConstants.NUM,
-                         vcf_header_io.VcfParserHeaderKeyConstants.TYPE])
-
-
-class VcfHeaderDefinitions(object):
-  """Container for header definitions."""
-
-  def __init__(self, vcf_header=None):
-    # type: (VcfHeader) -> None
-    """Initializes a ``VcfHeaderDefinitions`` object.
-
-    Creates two dictionaries (for infos and formats respectively) that map field
-    id to a dictionary which maps ``Definition`` to a list of file names.
-    """
-    self._infos = collections.defaultdict(dict)
-    self._formats = collections.defaultdict(dict)
-    if not vcf_header:
-      return
-    for key, val in vcf_header.infos.iteritems():
-      definition = Definition(
-          val[vcf_header_io.VcfParserHeaderKeyConstants.NUM],
-          val[vcf_header_io.VcfParserHeaderKeyConstants.TYPE])
-      self._infos[key][definition] = [vcf_header.file_name]
-    for key, val in vcf_header.formats.iteritems():
-      definition = Definition(
-          val[vcf_header_io.VcfParserHeaderKeyConstants.NUM],
-          val[vcf_header_io.VcfParserHeaderKeyConstants.TYPE])
-      self._formats[key][definition] = [vcf_header.file_name]
-
-  def __eq__(self, other):
-    return self._infos == other._infos and self._formats == other._formats
-
-  @property
-  def infos(self):
-    return self._infos
-
-  @property
-  def formats(self):
-    return self._formats
-
-
-class _DefinitionsMerger(object):
-  """Class for merging two ``VcfHeaderDefinitions``s."""
-
-  # For the same field definition, save at most `_MAX_NUM_FILE_NAMES` names.
-  _MAX_NUM_FILE_NAMES = 5
-
-  def merge(self, first, second):
-    # type: (VcfHeaderDefinitions, VcfHeaderDefinitions) -> None
-    """Updates ``first``'s definitions with values from ``second``."""
-    if (not isinstance(first, VcfHeaderDefinitions) or
-        not isinstance(first, VcfHeaderDefinitions)):
-      raise NotImplementedError
-    self._merge_definitions(first.infos, second.infos)
-    self._merge_definitions(first.formats, second.formats)
-
-  def _merge_definitions(
-      self,
-      first,  # type: Dict[str, Dict[Definition, List[str]]]
-      second  # type: Dict[str, Dict[Definition, List[str]]]
-      ):
-    # type: (...) -> None
-    """Updates ``first`` by merging values from ``first`` and ``second``."""
-    for key, definitions_to_files_map in second.iteritems():
-      for definition, file_names in definitions_to_files_map.iteritems():
-        first[key].setdefault(definition, [])
-        first[key][definition].extend(str(s) for s in file_names)
-        first[key][definition] = (
-            first[key][definition][:self._MAX_NUM_FILE_NAMES])
+from gcp_variant_transforms.beam_io import vcf_header_io # pylint: disable=unused-import
+
```

Comment:
nit: remove the extra line between the two imports. The general guide is that the imports are grouped in the order:
- standard library
- third party library
- local imports.
between each group we put one blank line.
read more: https://www.python.org/dev/peps/pep-0008/#imports